ecl/hql/hqlexpr.cpp

```diff
@@ -8147,6 +8194,91 @@ IHqlExpression *CHqlRemoteScope::clone(HqlExprArray &newkids)
     return this;
 }
 
+IHqlExpression * CHqlRemoteScope::lookupNearestSymbol(IIdAtom * searchName, unsigned lookupFlags, HqlLookupContext & ctx, NearestSymbol & nearest)
+{
+    preloadSymbols(ctx, false);
+
+    unsigned penalty = 0;
+    NearestSymbol tempNearest(searchName);
```

Comment:
I don't think there is any need to check the resolved member - all the information is in the base implementation.